documentation/source/install.rst

```diff
@@ -0,0 +1,114 @@
+************
+Installation
+************
+
+Using cocotb requires installation of pre-requisites and installation of cocotb itself.
+In this tutorial we are assuming that you already have a :ref:`supported simulator<simulator-support>` in ``PATH``.
+
+Pre-requisites
+==============
+
+Cocotb has the following requirements:
+
+* Python 3.5+
+* Python-dev packages
+* GCC 4.8.1+ or Clang 3.3+ and associated development packages
+* GNU Make
+* A Verilog or VHDL simulator, depending on your RTL source code
+
+.. versionchanged:: 1.4 Dropped Python 2 support
+
+.. note:: In order to use a 32-bit simulator you need to use a 32-bit version of Python.
+
+
+Linux - Debian/Ubuntu-based Systems
+-----------------------------------
+
+In a terminal, run
+
+.. code-block:: bash
+
+    sudo apt-get install make gcc g++ python3 python3-dev python3-pip
+    sudo apt-get install swig  # optional, needed for one of the examples
+
+.. seealso::
+
+   For more installation options, please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
+
+
+Linux - Red Hat-based Systems
+-----------------------------
+
+In a terminal, run
+
+.. code-block:: bash
+
+    sudo yum install make gcc gcc-c++ libstdc++-devel python3 python3-devel python3-pip
+    sudo yum install swig  # optional, needed for one of the examples
+
+.. seealso::
+
+   For more installation options, please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
+
+
+Windows
+-------
+
+`Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for Windows.
```

Comment:
Maybe `conda` should be first (possibly easiest way, also does not need any root access) for all OSes.